### DISCUSSION TITLE
Adapter setGalleryLayout function

### DISCUSSION THREAD

**Discussion by lovedota** [Azure/communication-ui-library#5228] [General]
Hi

I plan to subscribe to the event isLocalScreenSharingActiveChanged to change the View Layout based on that. Do we have the api 
`setGalleryLayout` available yet ?
```
     adapter.on('isLocalScreenSharingActiveChanged', (e) => {
        if (e.isScreenSharingOn) {
          adapter.setGalleryLayout('speaker')
        } else {
          adapter.setGalleryLayout('focusContent')
       }
      });
```

**Comment by JamesBurnside** [comment]
Hi @lovedota, we'll take a look into the available options we currently expose through the Composite options and get back to on this as soon as possible
